Description: Makefile: Make test target work during building the package.
 This patch changes the test directory paths, creates the directories,
 removes them after the test and adds a value to the argument -r in
 order to make the test target work in during dpkg-buildpackage.
Author: Martin Steigerwald <ms@teamix.de>

Index: fsmark/Makefile
===================================================================
--- fsmark.orig/Makefile	2012-04-26 16:42:14.756815557 +0200
+++ fsmark/Makefile	2012-04-26 16:46:39.328910106 +0200
@@ -6,8 +6,8 @@
 # Modifications include using fsync after wrting to flush to disk and changes to check return
 # values from syscalls.
 #
-DIR1= /test/dir1
-DIR2= /test/dir2
+DIR1= test/dir1
+DIR2= test/dir2
 
 COBJS= fs_mark.o lib_timing.o
 CFLAGS= -O2 -Wall -D_FILE_OFFSET_BITS=64
@@ -26,10 +26,12 @@
 	install ./fs_mark $(BIN)
 
 test: fs_mark
+	mkdir -p ${DIR1} ${DIR2}
 	./fs_mark -d ${DIR1} -d ${DIR2} -s 51200 -n 4096
-	./fs_mark -d ${DIR1} -d ${DIR2} -s 51200 -n 4096 -r 
+	./fs_mark -d ${DIR1} -d ${DIR2} -s 51200 -n 4096 -r 20
 	./fs_mark -d ${DIR1} -d ${DIR2} -s 51200 -n 4096 -D 128
-	./fs_mark -d ${DIR1} -d ${DIR2} -s 51200 -n 4096 -r -D 128
+	./fs_mark -d ${DIR1} -d ${DIR2} -s 51200 -n 4096 -r 20 -D 128
+	rm -rf ${DIR1} ${DIR2}
 
 clean:
 	rm -f ${COBJS} fs_mark fs_log.txt
